src/main/java/org/killbill/billing/plugin/adyen/core/resources/AdyenServlet.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2014-2017 Groupon, Inc
+ * Copyright 2014-2017 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.adyen.core.resources;
+
+import javax.inject.Singleton;
+
+import org.jooby.mvc.Path;
+
+@Singleton
+@Path("/")
+public class AdyenServlet {
+}
```

Comment:
We had it originally back in 2014 to debug some TermUrl stuff, but it hasn't been used for a while AFAIK (and it isn't super useful, see https://github.com/killbill/killbill-adyen-plugin/blob/6192e3e2d5fcf69c4d1855cfddf3ffb86b0e4b8b/src/main/java/org/killbill/billing/plugin/adyen/core/AdyenServlet.java).

However, we've recently talked about exposing more functionality (for production debugging and CS), similar to what was done in the PayPal plugin. So I figured I set up the wiring, so that it's easier for the next person to fill-in the blanks.